Makefile

```diff
@@ -1,17 +1,21 @@
-all: lkmpg.tex
-	rm -rf _minted-main
-	pdflatex -shell-escap lkmpg.tex
-	bibtex main >/dev/null || echo
+PROJ = lkmpg
+all: $(PROJ).pdf
+
+$(PROJ).pdf: lkmpg.tex
+	rm -rf _minted-$(PROJ)
```

Comment:
This directory `_minted-lkmpg` will be deleted in `clean` target in Makefile. 
Makefile should either delete this directory in `all` target or `clean` target,
but not both.